cpp/src/arrow/ipc/reader.cc

```diff
@@ -19,6 +19,7 @@
 
 #include <cstdint>
 #include <cstring>
+#include <iostream>
```

Comment:
Is this header actually used?

Reply:
No, I used it for debugging, and it should be removed.